diff --git a/data/net.supertuxkart.SuperTuxKart.metainfo.xml b/data/net.supertuxkart.SuperTuxKart.metainfo.xml
index baea4df..2007b0b 100644
--- a/data/net.supertuxkart.SuperTuxKart.metainfo.xml
+++ b/data/net.supertuxkart.SuperTuxKart.metainfo.xml
@@ -434,6 +434,9 @@
     <content_attribute id="social-chat">intense</content_attribute>
   </content_rating>
   <releases>
+    <release version="1.5" date="2025-10-20">
+      <url>https://blog.supertuxkart.net/2025/10/supertuxkart-15-release.html</url>
+    </release>
     <release version="1.4" date="2022-10-31">
       <url>https://blog.supertuxkart.net/2022/11/supertuxkart-14-release.html</url>
     </release>
diff --git a/data/po/update_desktop_file_appdata.py b/data/po/update_desktop_file_appdata.py
index d561909..278bea4 100755
--- a/data/po/update_desktop_file_appdata.py
+++ b/data/po/update_desktop_file_appdata.py
@@ -130,6 +130,9 @@ STK_APPDATA_FILE_5 = """  </description>
     <content_attribute id=\"social-chat\">intense</content_attribute>
   </content_rating>
   <releases>
+    <release version="1.5" date="2025-10-20">
+      <url>https://blog.supertuxkart.net/2025/10/supertuxkart-15-release.html</url>
+    </release>
     <release version="1.4" date="2022-10-31">
       <url>https://blog.supertuxkart.net/2022/11/supertuxkart-14-release.html</url>
     </release>
