diff -up fribidi/configure.ac.autoxxx fribidi/configure.ac
--- fribidi/configure.ac.autoxxx	2017-11-26 20:08:15.729211631 +0000
+++ fribidi/configure.ac	2017-11-26 20:30:00.418422977 +0000
@@ -122,7 +122,7 @@ AC_CHECK_SIZEOF(wchar_t)
 # Checks for compiler characteristics.
 changequote(,)dnl
 if test "x$GCC" = "xyes"; then
-  CFLAGS="$CFLAGS -Wall -ansi "
+  CFLAGS="$CFLAGS -ansi -Wno-deprecated-declarations"
 fi
 changequote([,])dnl

