diff -up fribidi/Makefile.am.autoxxx fribidi/Makefile.am
--- fribidi/Makefile.am.autoxxx	2017-11-26 21:00:35.023279284 +0000
+++ fribidi/Makefile.am	2017-11-26 21:00:45.287315233 +0000
@@ -11,7 +11,7 @@
 ACLOCAL_AMFLAGS = -I m4
 
 ## The order of subdirs is important, don't change without a reason.
-SUBDIRS = gen.tab charset lib bin doc test
+SUBDIRS = gen.tab charset lib bin test
 
 EXTRA_DIST = bootstrap ChangeLog.old

